### ISSUE TITLE
Why use value of “logvar” in model outputs as the self.adata.obsm['emb']？

### ISSUE THREAD

**Issue by StackChan** [ZhiWeiZhang0336/GraphCVAE#2]
https://github.com/ZhiWeiZhang0336/GraphCVAE/blob/6cad9ede9321fd4de727c3f5ae27363c0aa9e3e4/GraphCVAE/GraphCVAE.py#L267
```
            mu, logvar, hiden_feat, emb, ret, ret_a = self.model(self.features, features_a)
```
...
...
https://github.com/ZhiWeiZhang0336/GraphCVAE/blob/6cad9ede9321fd4de727c3f5ae27363c0aa9e3e4/GraphCVAE/GraphCVAE.py#L299

```
                   self.emb_rec = self.model(self.features, self.features_a)[1].detach().cpu().numpy()
                self.adata.obsm['emb'] = self.emb_rec
```

`self.emb_rec = self.model(self.features, self.features_a)[1].detach().cpu().numpy()`, [1] instead [3] , which means `logvar` instead of `emb`.
Why you assign value of `logvar` to self.emb_rec?

**Comment by ZhiWeiZhang0336** [comment]
I sincerely apologize for this oversight on our part. Thank you for pointing out this issue! We have already modified the code to correctly use index [3] to retrieve `emb` instead of incorrectly using `logvar`. Thank you again for your help.
